src/sibyl_utils.erl

```diff
@@ -254,9 +271,8 @@ grpc_port(PubKeyBin) ->
     Aliases = application:get_env(sibyl, node_grpc_port_aliases, []),
     case lists:keyfind(MAddr, 1, Aliases) of
         false ->
-            {ok, [GrpcOpts]} = application:get_env(grpcbox, servers),
-            #{listen_opts := #{port := Port}, transport_opts := #{ssl := SSL}} = GrpcOpts,
-            {Port, SSL};
+            {ok, Port} = application:get_env(miner, grpc_port, 8080),
```

Comment:
looks like @evanmcc put this under the `sibyl` app config in miner fwiw; also there's no wrapping `{ok, _}` tuple with `get_env/3`, only when you use the `get_env/2` variant

Reply:
fixed